{
  "uid" : "9ddad1df97782d89",
  "name" : "Verifying the backup mechanism -- @1.1 ",
  "fullName" : "Backup, Restore, and Update Mechanisms: Verifying the backup mechanism",
  "historyId" : "f8378de211ef0fdebac4909979724d24",
  "time" : {
    "start" : 1746710231798,
    "stop" : 1746710232367,
    "duration" : 569
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: STEP: When the admin initiates a system backup\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_006_steps.py\", line 14, in step_when_admin_initiates_backup\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: STEP: When the admin initiates a system backup\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_006_steps.py\", line 14, in step_when_admin_initiates_backup\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the admin user is logged into TPC_Web",
      "time" : {
        "start" : 1746710231799,
        "stop" : 1746710232340,
        "duration" : 541
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When the admin initiates a system backup",
      "time" : {
        "start" : 1746710232340,
        "stop" : 1746710232341,
        "duration" : 1
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: STEP: When the admin initiates a system backup\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_006_steps.py\", line 14, in step_when_admin_initiates_backup\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Then the system confirms the backup process completes successfully",
      "time" : {
        "start" : 1746710232367,
        "stop" : 1746710232367,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And the backup file backup_20250128 is retrievable and valid",
      "time" : {
        "start" : 1746710232367,
        "stop" : 1746710232367,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Backup, Restore, and Update Mechanisms"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "backup_file",
    "value" : "backup_20250128"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 2,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "f13f055e3a00ab43",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/20/#testresult/f13f055e3a00ab43",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: When the admin initiates a system backup\n",
        "time" : {
          "start" : 1746709777933,
          "stop" : 1746709778500,
          "duration" : 567
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "9ddad1df97782d89.json",
  "parameterValues" : [ "backup_20250128" ]
}